packages/newman-reporter-allure/src/index.ts

```diff
@@ -333,8 +336,32 @@ class AllureReporter {
     const requestData = this.currentRunningItem?.pmItem.requestData;
     const requestDataURL = requestData && `${requestData.method} - ${requestData.url}`;
 
+    if (requestData?.header && requestData?.header?.count() > 0) {
```

Comment:
No need to keep the left part of the expression (`requestData?.header`) because you're using optional chaining operator what already does the same

Reply:
Sorry, but no matter what I tried, I always got an error that the object is possibly undefined.

Reply:
Have you tried to add `?.` before the parens (`header?.count?.()`)?

Reply:
Yes. 
![image](https://github.com/allure-framework/allure-js/assets/91063753/56980bde-cf3c-4fb6-bca9-bbc40a59239a)

It is using workspace typescript (5.2.2-sdk). If I change it to VSCode's Version (5.2.2) the error in the editor is gone. But it still won't compile.

Reply:
Ok, no problem, let's leave it as is